src/js/components/Alert.js

```diff
@@ -3,9 +3,10 @@ import React from 'react';
 
 import Icon from './Icon';
 
-const Alert = ({children, showIcon, type}) => {
+const Alert = ({children, flushBottom, showIcon, type}) => {
   const classes = classNames('alert', {
-    [`alert-${type}`]: type != null
+    [`alert-${type}`]: type != null,
+    'flush-bottom': flushBottom === true
```

Comment:
1. Yes, that is the point. a `type` of 'foo' will add the class 'alert-foo'. Not sure I understand your comment? The PropTypes will make sure it is either type: 'danger' or 'success'. Also, I did not change this part.
2. Truthy value for `flushBottom` will not render `flush-bottom`, only `true` will apply the class: `1 === true` will equal `false`